src/components/QuestionPlay/WithOptions.tsx

```diff
@@ -1,54 +1,100 @@
 import React from 'react';
-import { Title, Button, Checkbox } from '@mantine/core';
+import { Title, Button, Checkbox, Group, createStyles } from '@mantine/core';
 import { Option as IOption } from '../../types';
 import Markdown from '../Markdown';
-import styles from './styles.module.css';
 
 interface Props {
-  setSelectedChoice: Function;
-  selectedOptionId: number | null | string;
+  setSelectedChoices: Function;
+  selectedOptionIds: number[] | null;
   options: IOption[];
-  selectedChoice: number | null | string;
+  selectedChoices: number[] | null;
   isAttempted: boolean;
   isTimerRunning: Boolean;
 }
 
+const useStyles = createStyles((theme) => ({
+  input: {
+    backgroundColor: 'rgb(255, 198, 198)',
+    borderColor: theme.colors.red,
+  },
+}));
+
 export default function WithOptions({
   options,
-  setSelectedChoice,
-  selectedOptionId,
-  selectedChoice,
+  setSelectedChoices,
+  selectedOptionIds,
+  selectedChoices,
   isAttempted,
   isTimerRunning,
 }: Props) {
+  const { classes } = useStyles();
+
+  function getCheckboxColor(option) {
+    if (selectedOptionIds && option.isCorrect && selectedOptionIds.includes(option.optionId)) {
+      return 'green';
+    } else if (
+      selectedOptionIds &&
+      ((!option.isCorrect && selectedOptionIds.includes(option.optionId)) ||
+        (option.isCorrect && !selectedOptionIds.includes(option.optionId)))
+    ) {
+      return 'red';
+    } else {
+      return '';
+    }
+  }
+
   return (
     <>
-      <Checkbox.Group
-        value={selectedChoice === null ? undefined : [selectedChoice.toString()]}
-        orientation="vertical"
-        label={<Title order={6}>Options</Title>}
-        size="md"
-        my="lg"
-        onChange={(value: string[]) => setSelectedChoice(value[value.length - 1])}>
-        {options.map((option) => (
-          <Checkbox
-            key={option.optionId}
-            label={<Markdown>{option.text}</Markdown>}
-            className={classNames({
-              [styles.isAttempted]: isAttempted,
-              [styles.correct]:
-                option.isCorrect && selectedOptionId && parseInt(selectedOptionId.toString()) === option.optionId,
-              [styles.inCorrect]:
-                !option.isCorrect && selectedOptionId && parseInt(selectedOptionId.toString()) === option.optionId,
-            })}
-            value={option.optionId.toString()}
-            disabled={isAttempted || !isTimerRunning}
-            radius="xl"
-            size="lg"
-          />
-        ))}
-      </Checkbox.Group>
+      <Group align="flex-start">
+        {isAttempted && (
+          <Checkbox.Group
+            value={options.filter((x) => x.isCorrect).map((x) => x.optionId.toString())}
+            orientation="vertical"
+            label={<Title order={6}>ANSWER</Title>}
+            size="md"
+            my="lg">
+            {options.map((x) =>
+              x.isCorrect ? (
+                <Checkbox
```

Comment:
This should be disabled as well?

Reply:
disabled checkbox doesn't show the background color, so using `pointerEvents: 'none'`
<img width="318" alt="Screenshot 2023-05-26 at 12 40 02 PM" src="https://github.com/rajatkantinandi/quizmaster/assets/14038952/bfc4a1aa-d99b-4ff8-87c1-52a831ce18a6">

Reply:
You can fix that with css. but should use disabled prop otherwise this is still clickable on touch screen and with keyboard.